media-gfx/apngopt/apngopt-1.4.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Optimizes APNG animations"
+DESCRIPTION="apngopt is a small utility which optimizes existing APNG animation files."
+HOMEPAGE="https://sourceforge.net/projects/apng/"
+COPYRIGHT="2011-2015 Max Stepin"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/apng/files/APNG_Optimizer/$portVersion/apngopt-$portVersion-src.zip"
+CHECKSUM_SHA256="c96047a893645fbd4afa5631b5d1728db3e6874ba8d76637de34355294a38bcf"
+SOURCE_DIR=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+
+PROVIDES="
+	apngopt$secondaryArchSuffix = $portVersion
+	cmd:apngopt = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
```

Comment:
We should use libpng 1.6.x instead of 1.5.x. Please replace libong by libpng16:
```diff
-	lib:libpng$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
```